src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IConversionExpression.vb

```diff
@@ -2348,24 +2349,81 @@ IVariableDeclarationStatement (1 declarations) (OperationKind.VariableDeclaratio
             VerifyOperationTreeAndDiagnosticsForTest(Of LocalDeclarationStatementSyntax)(source, expectedOperationTree, expectedDiagnostics)
         End Sub
 
+        <Fact()>
+        Public Sub ConversionExpression_Implicit_WideningDelegateTypeConversion()
```

Comment:
>ConversionExpression_Implicit_WideningDelegateTypeConversion [](start = 19, length = 60)

It would be good to cover an Anonymous Delegate conversions as well.

Reply:
Added.

---
In reply to: [138743990](https://github.com/dotnet/roslyn/pull/22056#discussion_r138743990) [](ancestors = 138743990)